src/TrackElementManager.jsx

```diff
@@ -81,15 +77,22 @@ export default class TrackElementManager extends React.Component {
             onCloseClick={this.onDeleteCloseClick.bind(this)}
             onConfirmClick={this.onDeleteConfirmClick.bind(this)} />
     </div>
-    
+
     <div className="clearfix"></div>
             </div>;
         } else {
             return <div></div>;
         }
     }
     onTextChange(e) {
-        this.props.onChange(this.props.activeElement, {source: e.target.value});
+        this.props.onChange(this.props.activeElement, {
+            source: e.target.value
+        });
+        jQuery('.jux-updated-text').show(0, function() {
```

Comment:
As far as which path to take, I can't decide, I'm pretty much indifferent. I will ask Zarina.